packages/php/email-editor/src/Utils/CdnAssetUrl.php

```diff
@@ -0,0 +1,20 @@
+<?php // phpcs:ignore SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing
+
+namespace MailPoet\EmailEditor\Utils;
+
+class CdnAssetUrl {
+  const CDN_URL = 'https://ps.w.org/mailpoet/';
+  /** @var string */
+  private $baseUrl;
+
+  public function __construct(
+    string $baseUrl
+  ) {
+    $this->baseUrl = $baseUrl;
+  }
+
+  public function generateCdnUrl($path) {
+    $useCdn = defined('MAILPOET_USE_CDN') ? MAILPOET_USE_CDN : true;
+    return ($useCdn ? self::CDN_URL : $this->baseUrl . '/plugin_repository/') . "assets/$path";
+  }
+}
```

Comment:
This is ok to copy now. But, IMO, it is something we should not use in this package as this is dependent on plugin assets. I'm not 100% sure where it is used. I assume it is in a pattern for basic content or perhaps a template. So later we need to move those to MailPoet plugin space.

Reply:
This is something I would like to address in a follow-up ticket. I will create it before moving this PR forward.